### PR TITLE
Issue #15449: excluded rule461verticalwhitespace from google-java-format.sh

### PR DESCRIPTION
**Pull request by Zopsss**
#15449

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java [hunk 1]
```diff
@@ -92,11 +92,11 @@ public int compareTo(InputVerticalWhitespace obj) { // ok
...
-                int number = 0;
-                return 0;
-              }
-            };
+        new Class2() {
+          public int compareTo(InputVerticalWhitespace obj) { // ok
+            int number = 0;
+            return 0;
+          }
+        };
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java:100]: formatter changed indentation. Not sure why the indentation wasn't changed during "formatting inputs" issue

**Comment by Zopsss** [comment]
https://github.com/checkstyle/checkstyle/blob/b7240ddffe8dadccedef8478fdb395f46b15df7c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java#L19-L22

Formatter originally formatted the above code to:

```java// violation below ''package' should be separated from previous line.'
package com.google.checkstyle.test.chapter4formatting.rule461verticalwhitespace;

import java.util.concurrent.Callable; // violation ''import' should be separated from previous

// line.'

class InputVerticalWhitespace { // violation ''CLASS_DEF' should be separated from previous line.'
```

But I reverted it so we can show violations.

Note: In the above code:

```java
 import java.util.concurrent.Callable; // violation ''import' should be separated from previous line.' 
```

line was 102 characters long, we allow for only 100 characters per line but linelength module didn't gave warning for this line. This may be a defect in our Check. @romani can you confirm this please?

Also, formatter added blank lines every where it was required, for example:

https://github.com/checkstyle/checkstyle/blob/b7240ddffe8dadccedef8478fdb395f46b15df7c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java#L86-L94

it added blank line before:

```java
   Class2 anon = // violation ''VARIABLE_DEF' should be separated from previous line.' 
```
```java
 class Class2 { // violation ''CLASS_DEF' should be separated from previous line.' 
```

So we started loosing violations but we required these violations to test the rule properly so I reverted those changes.

**Comment by romani** [comment]
cut a message a bit, try to make it like:
` import java.util.concurrent.Callable; // violation 'should be separated from previous line.'`

**Comment by romani** [comment]
> So we started loosing violations but we required these violations to test the rule properly so I reverted those changes.

yes, original Input `rule461verticalwhitespace/InputVerticalWhitespace.java` should stay the same for this lines.
and we create `ule461verticalwhitespace/InputFormattedVerticalWhitespace.java` that is copy of `rule461verticalwhitespace/InputVerticalWhitespace.java` and you remove all violations comments that are not violation any more.

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java (hunk 1 above)
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputVerticalWhitespace.java:100]: this is exactly a reason why we do this effort now to let CI check formatting for each commit.

**Comment by romani** [comment]
@Zopsss , please fix this PR.

**Comment by Zopsss** [comment]
> @Zopsss , please fix this PR.

we cannot exclude this rule as test starts failing after formatting code, my bad I shouldn't have sent PR for this rule.

**Pull request closed without merging**